india_banking/overrides/payment_entry.py

```diff
@@ -0,0 +1,89 @@
+import frappe
+from erpnext.accounts.doctype.accounting_dimension.accounting_dimension import (
+	get_accounting_dimensions,
+)
+from erpnext.accounts.doctype.payment_entry.payment_entry import PaymentEntry
+from erpnext.accounts.party import get_party_bank_account
+from frappe import _
+
+
+class CustomPaymentEntry(PaymentEntry):
```

Comment:
As you're not overriding anything here, then this class and file is not required? Can you move the below code elsewhere and remove this class and file.